dash-pipeline/bmv2/dash_flow.p4

```diff
@@ -0,0 +1,60 @@
+#ifndef _DASH_FLOW_P4_
+#define _DASH_FLOW_P4_
+
+#include "dash_headers.p4"
+
+control Flow(inout headers_t hdr, inout metadata_t meta) {
+    action flow_table_action(
+        bit<32> table_size,
+        bit<32> table_expire_time,
+        bit<32> table_version,
+        bit<32> table_key_flag,
+        bit<1> table_tcp_track_state,
+        bit<1> table_tcp_reset_illegal) {
+    }
+
+    action flow_entry_action(
+        bit<32> flow_version,
+        bit<32> flow_protobuf,
+        bit<32> flow_bidirectional,
+        bit<32> flow_direction,
+        bit<32> flow_reverse_key,
+        bit<32> flow_policy_result,
+        bit<32> flow_dest_pa,
+        bit<32> flow_metering_class,
+        bit<32> flow_rewrite_info,
+        bit<32> flow_vendor_metadata) {
+    }
+
+    @SaiTable[name = "flow_table", api = "dash_flow", api_order = 0, isobject="true"]
+    table flow_table {
+        key = {
+            meta.eni_id : exact @SaiVal[type = "sai_object_id_t"];
```

Comment:
I think I found a problem if we put eni_id as an attribute to the flow table directly - it will be hard to support DPU-level bulk sync.

The reason is that with current approach, the flow table is bound to the ENI level - each ENI will have its own flow table. Hence, in order to implement DPU-level bulk sync, we will need to call bulk get API on all flow table - essentially use ENI-level bulk sync to implement DPU-level bulk sync. This might not be supported.

A better way to implement this would be:
1. Create flow table without ENI, but it can have all the attributes to set which field to enable in the flow keys, and ENI is one of them.
2. Add flow table id on the ENI, so we can bind the flow table on all ENIs (DPU-level) or bind the flow table to a single ENI (ENI-level). In the future, we can even bind multiple flow table to a ENI, which is very flexiable.

With this, to implement DPU-level bulk sync, all we need is to bulk get all flow entries on the DPU-level flow table, and call it done.